rotkehlchen/db/schema.py

```diff
@@ -313,11 +313,18 @@
 );
 """  # noqa: E501
 
-DB_CREATE_ETHEREUM_ACCOUNTS_DETAILS = """
-CREATE TABLE IF NOT EXISTS ethereum_accounts_details (
-    account VARCHAR[42] NOT NULL PRIMARY KEY,
-    tokens_list TEXT NOT NULL,
-    timestamp INTEGER NOT NULL
+
+# Store information about the tokens queried for each combination of account and chain. Previously
+# we were using a json and a tiemstamp column. Now the table is designed to have a key-value
+# structure where we use the key `token` to identify the tokens queried per address and the
+# key `last_queried_timestamp` to determine when the last query was executed
+DB_CREATE_EVM_ACCOUNTS_DETAILS = """
+CREATE TABLE IF NOT EXISTS evm_accounts_details (
+    account VARCHAR[42] NOT NULL,
+    chain CHAR(1) NOT NULL DEFAULT('A'),
```

Comment:
Is following the same we do in the globaldb so then we can use the deserialize from the `ChainID` class

Reply:
Okey, I have changed it to follow the pattern we use in the user db of text for the blockchain and it makes sense to use the supportedBlockchain enum instead of the chainid